install/helm/README.md

```diff
@@ -30,13 +26,12 @@ you can use the helm `--namespace` parameter to specify a different namespace._
 The command deploys Agones on the Kubernetes cluster with the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
 > **Tip**: List all releases using `helm list`
->
+
 
 > If you are installing a development build of Agones (i.e. not the 0.4.0 release), you will need to install Agones the following way:
 
 ```bash
-$ cd install/helm/
-$ helm install --name my-release --namespace agones-system agones --set agones.image.tag=0.4.0-481970d
+$ helm install --name my-release --namespace agones-system agones/agones --set agones.image.tag=0.4.0-481970d
```

Comment:
Should we leave this as "cd install/helm" - just because if you are installing the development build, you likely want to do it from github source (which we should probably also be clearer), as the install scripts for development may be different than what was in the previous release. WDYT?

Reply:
For now, should we make it a `git clone` then `cd` into the directory operation?

Reply:
ignore the above. I missed this was outdated.